Shared/sdk/net/bitstream.h

```diff
@@ -548,6 +548,10 @@ enum class eBitStreamVersion : unsigned short
     // 2024-05-23
     WorldSpecialProperty_ExtendedWaterCannons,
 
+    // Add "extendedwatercannons" to setWorldSpecialPropertyEnabled
+    // 2024-06-30
+    WorldSpecialProperty_TunnelWeatherBlend,
+
```

Comment:
Looks like the only mistake here: Move this below `PedSync_Revision`

Reply:
All right nice hit!